src/estimators/any_sketch.py

```diff
@@ -317,8 +353,18 @@ def add(self, x):
     indexes = self.get_indexes(x)
     # Move this to the loop when we support more than one value function.
     value_function = self.config.value_functions[0]
+
+    if value_function == UniqueKeyFunction():
+      assert self.config.num_hashes == 1, (
```

Comment:
Why have this limitation?

Reply:
Removed this in the updated version.

Was adding this limitation because I could not think of an estimator for SKA with multiple hashes...